pom.xml

```diff
@@ -477,7 +487,7 @@
 
     <properties>
         <!--Identity Governance Version-->
-        <identity.governance.version>1.11.11</identity.governance.version>
+        <identity.governance.version>1.11.18</identity.governance.version>
```

Comment:
Bump governance version once https://github.com/wso2-extensions/identity-governance/pull/887 is merged.

Reply:
Bumped with https://github.com/wso2-extensions/identity-event-handler-notification/pull/291/commits/1eb1c1eabcfc569b87195d0f63132736d3ccd55f